cpu/cpu.go

```diff
@@ -40,12 +40,18 @@ type InfoStat struct {
 	CoreID     string   `json:"coreId"`
 	Cores      int32    `json:"cores"`
 	ModelName  string   `json:"modelName"`
-	Mhz        float64  `json:"mhz"`
+	Mhz        Mhz      `json:"mhz"`
 	CacheSize  int32    `json:"cacheSize"`
 	Flags      []string `json:"flags"`
 	Microcode  string   `json:"microcode"`
 }
 
+type Mhz struct {
```

Comment:
I would use MHz

And while you are changing the type and so the output format, I would change the Mhz variable name in the struct

But here,I'm only a random Gopher.

The maintainers of this project might have another opinion